crates/rattler_installs_packages/src/index/direct_url/file.rs

```diff
@@ -47,7 +47,7 @@ pub(crate) async fn get_sdist_from_file_path(
 
     let file = File::open(path).into_diagnostic()?;
 
-    let dummy_sdist = SDist::from_bytes(dummy_sdist_file_name, Box::new(file))?;
+    let mut dummy_sdist = SDist::from_bytes(dummy_sdist_file_name, Box::new(file))?;
```

Comment:
I would not call it `dummy` because after you set the name its `real` however, the file name is a dummy though.